scheduler_test.go

```diff
@@ -82,50 +83,57 @@ func TestStartImmediately(t *testing.T) {
 }
 
 func TestAt(t *testing.T) {
-	s := NewScheduler(time.UTC)
+	t.Run("runAt(3 seconds in the future) should run today at current time plus 3 seconds", func(t *testing.T) {
```

Comment:
the test below is called `TestAtFuture` - this should either move there, or could we combine these into a single test and make it a table test?

Reply:
That makes sense, will take a look at it later tonight